components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/user/user-mgt.jsp

```diff
@@ -748,35 +748,6 @@
                 }
             %>
 
-            <%
-                if ((multipleUserStores || !userRealmInfo.getPrimaryUserStoreInfo().getReadOnly())
```

Comment:
Check if this variable multipleUserStore is used anywhere before or after this section. If not this is unused and you must remove it.